app/src/main/java/com/codesoom/assignment/application/ProductService.java

```diff
@@ -53,6 +77,12 @@ public Product deleteProduct(Long id) {
         return product;
     }
 
+    /**
+     * id에 해당하는 상품을 찾아 리턴한다.
+     * @param id
+     * @return 찾은 상품 정보
+     * @throws id에 해당하는 상품을 찾지 못할 경우 ProductNotFoundException
```

Comment:
앞뒤가 바뀌어 있어요. 예외 타입이 먼저 와야 합니다.
```suggestion
     * @throws ProductNotFoundException id에 해당하는 상품을 찾지 못할 경우
```